infra/build_specified_commit.py

```diff
@@ -144,17 +143,10 @@ def copy_src_from_docker(project_name, host_dir):
   return src_dir
 
 
+@retry.wrap(_IMAGE_BUILD_TRIES, 2, 'infra.build_specified_commit')
```

Comment:
infra.build_specified_commit. _build_image_with_retries for function name

Reply:
Done.